README.md

```diff
@@ -1,24 +1,238 @@
-# README
+<a name="readme-top"></a>
```

Comment:
## I couldn't find an open PR on the frontend project, therefore, I'll leave my comments here:

- [ ] For your next review request, kindly open a PR on the frontend project.   It should contain all the content and point from dev to main so a proper review can be carried out.
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.
- [ ] `App.test.js` contains a test that is unused.  It is the default one that comes with CRA.  Kindly delete the file if you are not using it.
- [ ] In `reservations.js` kindly remove the old piece of commented code, to improve your repo's cleanliness.
- [ ]  Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.
![image](https://user-images.githubusercontent.com/57421823/206490388-b1fbc1b8-4a3d-49f9-9d44-2feea066a69c.png)

If you follow these steps, including the open PR on your frontend, your project will be ready. 💪